src/main/java/com/checkout/disputes/four/DisputesClient.java

```diff
@@ -3,18 +3,20 @@
 import com.checkout.common.FileDetailsResponse;
 import com.checkout.common.FileRequest;
 import com.checkout.common.IdResponse;
+import com.checkout.disputes.DisputeEvidenceRequest;
+import com.checkout.disputes.DisputeEvidenceResponse;
 
 import java.util.concurrent.CompletableFuture;
 
 public interface DisputesClient {
 
     CompletableFuture<DisputesQueryResponse> query(DisputesQueryFilter queryFilter);
 
-    CompletableFuture<DisputeDetailsResponse> getDisputeDetails(String disputeId);
+    CompletableFuture<DisputeDetailsResponse> disputeDetails(String disputeId);
```

Comment:
Why the different name? It's not possible to override the method from the parent?